app/assets/stylesheets/govuk-component/govspeak/_example.scss

```diff
@@ -0,0 +1,11 @@
+// Scoped to govspeak container when included
```

Comment:
Is it worth explaining what each feature is used for at the top of each file?

I'm not sure what `.example` is for, for instance.